### PR TITLE
Unit02_02-Many-to-many_Andre_Part1_answers (worked with Darien)

### PR DESCRIPTION
**Pull request by nakkurt**
##Unit02_02-Many-to-many_Andre_Part1_answers (worked with Darien):
This covers Part 1 of U2-02 - many to many. I worked with Darien on this Part 1.

### CODE REVIEW

File: Unit-02/02-many-to-many/project/departments/views.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        department_form = NewDepartmentForm(request.form)
+        if department_form.validate():
+            department.name = department_form.name.data
+            db.session.add(department)
+            db.session.commit()
+            return redirect(url_for('departments.index'))
+        return render_template('departments/edit.html', department=department, form=department_form)
+    return render_template('departments/show.html', department=department, delete_form = delete_form)
+
+@departments_blueprint.route('/new')
```
Comment (mmmaaatttttt) [inline Unit-02/02-many-to-many/project/departments/views.py:45]: small thing, but it's good to be consistent with white space. most of your routes have 2 lines between them, this one only has one.

File: Unit-02/02-many-to-many/project/employees/templates/employees/show.html [hunk 2]
```diff
@@ -0,0 +1 @@
+Implement me! The show page for employees
```
Comment (mmmaaatttttt) [inline Unit-02/02-many-to-many/project/employees/templates/employees/show.html:1]: haha, this show page looks like it could use some love!

**Review by mmmaaatttttt — COMMENTED** [review]
Hey @nakkurt - looks good, although your show page for users could use some love :)

Let me know when you've added tags to users/messages. You can either add to this PR or open a new one, whatever's easier. If you open a new one, i'll close this one.

**Comment by nakkurt** [comment]
> Let me know when you've added tags to users/messages.

thx @mmmaaatttttt for the code review.
Here's part 2 finished -> added tags to users-messages in [this PR](https://github.com/nakkurt/python_curriculum_exercises/tree/Unit02_02-Many-to-many_Andre_Part2_answers):

**Pull request closed without merging**